docs/user-guide/kubernetescurator.rst

```diff
@@ -385,3 +385,85 @@ To download data from your PVC, you can use the ``nemo-workspace-busybox`` Pod c
 
     # Copy directory in PVC /nemo-workspace/fizzbuzz to local file-system at $LOCAL_WORKSPACE/fizzbuzz
     kubectl exec nemo-workspace-busybox -- tar cf - /nemo-workspace/fizzbuzz | tar xf - -C $LOCAL_WORKSPACE
+
+
+Running existing Modules using Kubernetes Python client:
+
+------------------------------------------------------
+
+This approach involves programmatically establishing a connection to the scheduler pod in order to execute the current NeMo curator modules.
+
+.. code-block:: bash
+
+    # create a python environment
+    python3 -m venv venv
+    source venv/bin/activate
+    pip install kubernetes
```

Comment:
Combine this with the existing environment setup information, or omit the
```
python3 -m venv venv
source venv/bin/activate
```
Since it was already done.

Also, please clarify that only this option of doing things requires this pip package, and explain what it is used for.